src/test/groovy/org/prebid/server/functional/tests/storage/StorageBaseSpec.groovy

```diff
@@ -0,0 +1,44 @@
+package org.prebid.server.functional.tests.storage
+
+import org.prebid.server.functional.service.PrebidServerService
+import org.prebid.server.functional.service.S3Service
+import org.prebid.server.functional.testcontainers.PbsServiceFactory
+import org.prebid.server.functional.tests.BaseSpec
+import org.prebid.server.functional.util.PBSUtils
+
+class StorageBaseSpec extends BaseSpec {
+
+    protected static final String INVALID_FILE_BODY = 'INVALID'
+    protected static final def DEFAULT_BUCKET = PBSUtils.randomString.toLowerCase()
```

Comment:
Pretty minor, for consistency move `def` to `String`

Reply:
done